app/views/layouts/_user_config.html.erb

```diff
@@ -7,7 +7,9 @@
       document.body.className = bodyClass;
 
       if (bodyClass.includes('dark-theme')) {
-        document.getElementById('body-styles').innerHTML = '<style><%= Rails.application.assets["themes/dark.css"].to_s.squish.html_safe %></style>';
+        const styles = document.createElement('style');
+        styles.innerHTML = '<%= Rails.application.assets["themes/dark.css"].to_s.squish.html_safe %>';
+        document.getElementById('body-styles').append(styles);
```

Comment:
So far dark theme has been ignoring forem's brand color defined by admin. This will change it so dark theme will also reflect brand colors.